docs/event/metadata.md

```diff
@@ -4,38 +4,79 @@ sidebar_position: 2
 - `creators: NonBlankString[]`: The people mainly responsible for creating this video and/or presenting the talk which this video is a recording of. Should contain human-readable names and not usernames. Plain text. This is the main "who?"-information shown in the UIs; other fields in `extraMetadata` (e.g. `dct:contributor`) might be shown too, but less prominently.
 - `language: LangCode?`: describes the (main) language of this event and its metadata. For example, the audio language and (if applicable) language of video content is more important than the language of available subtitles. Generally, assets can have their own language specified.
 - `series: SeriesID?`: optional ID of the series this event belongs to. Must be a valid series ID of an existing series at all time.
 - `owner: Username`: TODO figure out details
-
-#### Time-data
+- `createdBy: Username` 🟦: username of the user that created this event.
+  - Like `created`, this refers to the moment when the event is first added to the database, not necessarily when it is ingested.
+  - This refers to the user with which the API request is authorized, e.g. potentially an API user and not referring to an actual human person.
+  - Technical field, not intended to be shown to normal users.
+  - Set by Opencast and cannot be changed.
+  - When duplicating an event, the new event has this field set to the duplicating user.
+
+### Time-data
 - `startTime: DateTime?`: Actual real life datetime when the video recording started or will start, with timezone. If this is not applicable, for example because it's a short movie, this should be undefined. UIs should use this as primary date to show for a video and if unset, fallback to `created`.
 - `endTime: DateTime?`: Like `startTime`, but when the video recording stopped. Due to cutting, recording pauses and etc, the `duration` is not necessarily `end - start`.
 - `duration: Milliseconds` 🟦: duration of the event. As specified in ["assets"](./assets), this needs to always match the duration of all non-internal tracks.
-- `updated: Timestamp` 🟦: Timestamp of when anything about this event was last changed.
-- `created: Timestamp` 🟦: Timestamp of when the event was created in Opencast. It is set once when the event is first stored in Opencast's DB, and never changed again. This also implies that scheduled event's `created` date is when the scheduling took place, _not_ the time it is scheduled for (that would be `startDate`)
+- `modified: Timestamp` 🟦: Timestamp of when anything about this event was last changed.
+  - More precisely: at any point in time since `modified`, all fields, assets, ACL and any other part of the event data model need to have the exact same value as they have at the present moment.
+  Whenever anything about an event described in this data model changes, `modified` has to be set to `now()`.
+    - Noteworthy case: when a series is deleted and the event's `series` field is set to `null`, the event's `modified` needs to change.
```

Comment:
Do we still want this configurable? Currently, you can configure if deleting a non-empty series is forbidden or allowed and leaving the events without a series. I would suggest disallowing deleting non-empty series and making this not configurable.